python_modules/dagster-graphql/dagster_graphql/implementation/execution/backfill.py

```diff
@@ -153,6 +154,10 @@ def create_and_launch_partition_backfill(
                 "arguments"
             )
 
+        is_valid_title, reason = is_valid_title_and_reason(backfill_params.get("title"))
+        if not is_valid_title:
+            raise DagsterInvariantViolationError(reason)
```

Comment:
do i need to make my own `GrapheneInvalidTitleError` in order to make this actually fail instead of just setting `_PythonError` or do we have a general purpose class for raising errors? It's also unclear to me what our best practices are around what exceptions should be raised as Graphene errors vs piped through via _PythonError.

cc @alangenfeld

Reply:
this `@capture_error`
https://github.com/dagster-io/dagster/blob/master/python_modules/dagster-graphql/dagster_graphql/schema/roots/mutation.py#L321
willi take any exception thrown here and turn it in to a `PythonError`, unless there is a Graphene error type for it
https://github.com/dagster-io/dagster/blob/master/python_modules/dagster-graphql/dagster_graphql/implementation/utils.py#L157-L181

the custom Graphene error types just allow the front end to know what type of error it is and perform special handling of it. If the behavior is always "show dialog with error message" than `PythonError` is fine. 

If we wanted to instead have just the title box turn red and communicate the error more preciesely, then you need to add a new custom Graphene error object for the client to `__typename` match against and optionally any other special fields it might want for more information